sycl/test/on-device/online_compiler/online_compiler_common.hpp

```diff
@@ -0,0 +1,180 @@
+#include <CL/sycl.hpp>
+#include <CL/sycl/INTEL/online_compiler.hpp>
+
+#include <iostream>
+#include <vector>
+
+static constexpr char *CLSource = R"===(
```

Comment:
Probably the user do not really car about the exact type here either for all these sources...
```suggestion
auto constexpr CLSource = R"===(
```